llama-index-integrations/tools/llama-index-tools-jina/README.md

```diff
@@ -0,0 +1,27 @@
+# LlamaIndex Tools Integration: Jina
+
```

Comment:
nit: lets add the install here: `pip install llama-index-tools-jina` ?

Reply:
> nit: lets add the install here: `pip install llama-index-tools-jina` ?

Added.